_data/lang/en.yml

```diff
@@ -197,8 +197,16 @@ constants:
 
 # for contact form
 contact_form:
-  button_name: Contact Form
-  formLoadingText: Loading the Contact Form...
+  button_name: Submit
+  name: Name
+  email: Email
+  reason: Reason for contact
+  discuss: Discuss a project
```

Comment:
The reasons could be separated from the rest of variables like `offer`in `lng`:
https://github.com/curacultura/curacultura-website/blob/63c5a3c4603845af0e7b0217d519291f93845471/_data/lang/en.yml#L5-L18